drivers/ethernet/eth_stm32_hal_priv.h

```diff
@@ -38,6 +38,10 @@ struct eth_stm32_hal_dev_data {
 	ETH_HandleTypeDef heth;
 	/* clock device */
 	struct device *clock;
+#ifdef CONFIG_SOC_SERIES_STM32H7X
+	/* interrupt semaphores to wait for ETH transfer complete IT */
```

Comment:
Please delete this comment. It is obvious, what this semaphore do. I would also put this semaphore right after `rx_int_sem` (see line 46). Then it is clear - they are doing similar job.

Reply:
Done 👍